File: 99_bottles.js
function bottleSong(num) {
  for (let i = num; i > 2; i--) {
    console.log(i + ' bottles of beer on the wall, ' + i + ' bottles of beer.')
    console.log('Take one down and pass it around, ' + (i-1) + ' bottles of beer on the wall.') 
  }  
  console.log('2 bottles of beer on the wall, 2 bottles of beer.')
  console.log('Take one down and pass it around, 1 bottle of beer on the wall.')
  console.log('1 bottle of beer on the wall, 1 bottle of beer.')
  console.log('Take one down and pass it around, no more bottles of beer on the wall.')
  console.log('No more bottles of beer on the wall, no more bottles of beer.')
  console.log('Go to the store and buy some more, ' + num + ' bottles of beer on the wall.')
};


bottleSong(99);

File: 99_bottles.py
def bottle_song(num):
	while num > 2:
		print(f"{num} bottles of beer on the wall, {num} bottles of beer.\nTake one down and pass it around, {num - 1} bottles of beer on the wall.")
		num -= 1
	print(f"2 bottles of beer on the wall, 2 bottles of beer.\nTake one down and pass it around, 1 bottle of beer on the wall.\n1 bottle of beer on the wall, 1 bottle of beer.\nTake one down and pass it around, no more bottles of beer on the wall.\nNo more bottles of beer on the wall, no more bottles of beer.\nGo to the store and buy some more, 99 bottles of beer on the wall.")

bottle_song(99)